openml-lightgbm/make.sh

```diff
@@ -16,15 +16,28 @@
 #
 # @author Sheng Wang (sheng.wang@feedzai.com)
 
-FILE=lightgbmlib_build
-if [ ! -d "$FILE" ]; then
-  echo "entering the folder."
+VERSION=v"$1"
+
+# Compare version, when different remove the build folder
+FILE=lightgbmlib_build/__version__
+if [ -f "$FILE" ]; then
+  OLD_VERSION=$(head -n 1 "$FILE")
+  if [ "$VERSION" != "$OLD_VERSION" ]; then
+    echo "Renaming the folder."
+    rm -rf lightgbmlib_build
+  fi
+fi
+
+# Make LightGBM if it doesn't exist
+DIR=lightgbmlib_build
+if [ ! -d "$DIR" ]; then
+  echo "Entering the folder."
   cd make-lightgbm || return
```

Comment:
Remove all return, they are invalid.

Use `set -e` instead to abort at any error.